ravenframework/CodeInterfaceClasses/RELAP5/Relap5Interface.py

```diff
@@ -79,12 +79,20 @@ def _readMoreXML(self,xmlNode):
     self.inputAliases = {}
     self.outputDeck = -1 # default is the last deck!
     self.operators  = []
+    self.datatypes = {}
     for child in xmlNode:
       if child.tag == 'outputDeckNumber':
         try:
           self.outputDeck = int(child.text)
         except ValueError:
           raise ValueError("can not convert outputDeckNumber to integer!!!! Got "+ child.text)
+      elif child.tag == 'datatypes':
+        # DATA TYPES
+        integers = child.find("integers")
+        if integers is not None:
+          c = "," if "," in integers.text else None
+          self.datatypes['integers'] = [e.strip() for e in (integers.text.split() if c is None else integers.text.split(c))]
```

Comment:
only integers are processed here. I would suggest to add a check here, I would suggest to check all the subnodes under datatypes to prevent any typos or other types provided by users.

Reply:
completely right. Fixed